content/adventures/en.yaml

```diff
@@ -2531,6 +2531,304 @@ adventures:
                     ```
 
                     {print} 2.5 + 2.5
+    music:
+        name: music
+        default_save_name: music
+        description: Play a tune!
+        levels:
+            1:
+                story_text: |-
+                    In this level you'll learn how to use the `{play}` command to play a tune!
+
+                    Type `{play}` followed by the note you want to play. The scale goes C-D-E-F-G-A-B.
+                    As you can see there are 7 different letters, but we can play more than just 7 notes.
+                    Type a number between 1 and 10 behind the letter to choose the scale, for example after B4 comes C5.
+                    C1 is the lowest note you can play, C10 is the highest.
+
+                    ### Exercise
+                    Try out the example code and thenplay around with it! Can you create your own melody?
+                    In the next level you'll learn how to play some existing songs.
+                example_code: |-
+                    ```
+                    play C4
+                    play D4
+                    play E4
+                    play F4
+                    play G4
+                    play A4
+                    play B4
+                    play C5
+                    ```
+            2:
+                story_text: |-
+                    Finish the songs! We have started the codes for 3 melodies.
+                    As you can see, you can also use the `{sleep}` command to add a little pause in the song.
+                example_code: |
```

Comment:
Hi @MarleenGilsing!

I have added a story_text here, because if you have more example codes than story_texts they will not show. I have also removed 1 song, since 3 is a bit much and the block becomes long. Once we have shared adventures (very soon!!) we can add as much as we want :)